app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -11,7 +11,7 @@ navigation: 20.10
 navigation_title: Train to teach in England as a non-UK citizen
 navigation_description: Find out how to train to teach in England as a non-UK citizen or foreign student and get English qualified teacher status (QTS).
 date: "2021-05-27"
-image: "static/images/content/hero-images/0003.jpg"
+image: "static/images/content/hero-images/teacher3.jpg"
```

Comment:
I think we should maybe replace this with 0034.jpg as per the other pages @jenhadfield-dfe